### ISSUE TITLE
Update method not work

### ISSUE THREAD

**Issue by SmileGobo** [seveas/python-networkmanager#23]
If I try reset IP settings or wireles settings have error:
**according to signature 'a{sa{sv}}': <class 'ValueError'>: Unable to guess signature from an empty list**

```python
   #extract item from dict  NetworkManager.NetworkManager.Settings.Connection
   conn = self._list[name] 
   s = conn.GetSettings()
   RemoveKey(s, 'ipv6') #this method remove extra keys without error
   t = self._Type(name) #extract type from ['connection']['type']
   RemoveKey(s, t)
   RemoveKey(s, t + '-security')
   #just try reset settings 
   conn.Update(s) 
```

**Comment by seveas** [comment]
That's the same issue as #17

**Comment by seveas** [comment]
Fixed in 1.2
